README.md

```diff
@@ -1,2 +1,23 @@
 # ilib-loctool-arb
 Ilib loctool plugin to parse and localize ARB files
+
+v1.0.0
+* Implement for [ARB](https://github.com/google/app-resource-bundle/wiki/ApplicationResourceBundleSpecification) file type of localization.
+  *  It locaclize the `intl_messages.arb` file which is the result of `extract_to_arb.dart` program provided by [intl_translation](https://pub.dev/packages/intl_translation)
+
+## License
+
+Copyright (c) 2023, JEDLSoft
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
No version history. Should have v1.0.0 doc here at least

Reply:
I add some explanation. Please see if it's ok or need more information